.github/workflows/ci-renovate.yml

```diff
@@ -27,21 +27,36 @@ jobs:
           fetch-depth: 0
           # Checkout pull request HEAD commit instead of merge commit.
           ref: ${{ github.event.pull_request.head.sha }}
+          # Needed to enable checks re-run.
+          token: ${{ secrets.PAT }}
       - name: Set workflow variables
         # Id is needed to access output in a next step.
         id: vars
         run: |
           echo "FILE_NAME=.changelog/${{ github.event.pull_request.number }}.internal.md" >> $GITHUB_OUTPUT
+      - name: Check if Change Log file exists
+        id: check_file
+        run: |
+          if [[ ! -f "${{ steps.vars.outputs.FILE_NAME }}" ]]; then
```

Comment:
Ideally, this should check if `.changelog/${{ github.event.pull_request.number }}.*.md` exists.
If not, then create `{{}}.internal.md`